elements/lisk-transactions/src/utils/sign_and_validate.ts

```diff
@@ -41,98 +36,3 @@ export const validateSignature = (
 			: undefined,
 	};
 };
-
```

Comment:
Out of curiosity, why are these changes (and those deleted functions at `test/utils/sign_and_validate.spec.ts`) part of this PR? Should not these be part of improve multisignature set of changes?

Reply:
it was suppose to be cleaned up during multi-sign changes, since it was not i cleaned it up :P